### DISCUSSION TITLE
Very slow performance in larger playlists

### DISCUSSION THREAD

**Discussion by g4njawizard** [navidrome/navidrome#2322] [Q&A]
Hi everyone,

I don't have extremely large playlists or mass songs in general, but the performance of "larger" playlists is really affecting the joy of using the UI.
I have a playlist with around ~300 songs (3GB) and interacting with each song in terms of opening the context and moving these songs somewhere else or removing them takes between 5-30 seconds per interaction. After every interaction it takes some time reloading. That's a lot of time when I always have to move every single song by hand. Unfortunately there is no option to mark multiple songs inside a playlist and move/copy to different playlists.
Currently Navidrome is the only container running on my unraid server with 32GB and Ryzen 1700X CPU. So absolute overkill atm. My Client which I use for browsing is not bottlenecking in terms of performance.
Is this kind of behaviour expeted, or what exactly is slowing down the performance?
The Scan shedule is around 45min
Image cachesize= 2000MB
Transcodingcache 1500MB

Kind regards

**Comment by deluan** [comment]
The issue is in the web ui, not the server. This is caused by the way items are rendered in the list. Hopefully this will be better in the next version of the UI, coming up in the next months.
